### DISCUSSION TITLE
回声洞文件放在哪

### DISCUSSION THREAD

**Discussion by iPodtouch6557** [Meloong-Git/PCL#2563] [提问] [answered]
应该是像Minecraft那个splashes.txt一样的吧……就是不知道在哪个路径能看源文件

**Comment by WForst-Breeze** [comment]
之前翻这个仓库公开了的源码我自己好像没找到回声洞文件存在哪，源文件貌似也没在哪个路径找到过，应该是没公开？

**Comment by LTCatt** [comment, marked as answer]
回声洞的内容没有公开，抱歉（
